yt_dlp/extractor/nebula.py

```diff
@@ -76,8 +80,12 @@ def _build_video_info(self, episode):
         fmts, subs = self._fetch_video_formats(episode['slug'])
         channel_slug = episode['channel_slug']
         channel_title = episode['channel_title']
+        if episode['zype_id']:
+            zype_id = episode['zype_id']
+        else:
+            zype_id = None
         return {
-            'id': episode['zype_id'],
+            'id': episode['id'].replace('video_episode:', ''),
```

Comment:
This isn't absolutely necessary, but it's more precise (remove from the start) and would give us a more descriptive error if something goes wrong (`Extractor failed to obtain "id"` instead of `NoneType has no attr replace`)
```suggestion
            'id': remove_start(episode['id'], 'video_episode:'),
```